services/web-app/layouts/layout/layout.js

```diff
@@ -124,7 +124,7 @@ const Layout = ({ children }) => {
                           }
                           if (!data.path && data.items) {
                             return (
-                              <>
+                              <span key={i}>
```

Comment:
Can this be a `div` because some of the children like `h2` are display block?

Reply:
Done!